pkg/clients/gitlab/git.go

```diff
@@ -104,13 +109,20 @@ func (gc *GitlabClient) GetMergeRequests() ([]*gitlab.MergeRequest, error) {
 
 // CloseMergeRequest closes merge request in Gitlab repo by given MR IID
 func (gc *GitlabClient) CloseMergeRequest(projectID string, mergeRequestIID int) error {
-
 	// Get merge requests using Gitlab client
-	_, _, err := gc.client.MergeRequests.GetMergeRequest(projectID, mergeRequestIID, nil)
+	_, resp, err := gc.client.MergeRequests.GetMergeRequest(projectID, mergeRequestIID, nil)
 	if err != nil {
+		// Check if the error is due to the MR not existing
+		if resp != nil && resp.StatusCode == http.StatusNotFound {
+			// MR does not exist, log a message and return nil
+			fmt.Printf("Merge request with IID %d in projectID %s does not exist\n", mergeRequestIID, projectID)
+			return nil
```

Comment:
The returned error is `nil`, which will not fail the assertions in the e2e tests. Can we merge these 2 lines?

Reply:
this check is for the cases when the MR is not exist so there is no MR to close , in this case I want to print out to user about that yet not be failed in assertion if for some reason the MR was deleted or closed before .

Reply:
Maybe I'm wrong, but I'd expect the MR always to exist. If it doesn't exist, then that should be an error because the `GetMergeRequest()` function must always return the correct MR even if it's closed. Wdyt?

Reply:
agree, 
fixed the function as it was on the main

Reply:
thank you!